qa/L0_batcher/batcher_test.py

```diff
@@ -77,6 +77,7 @@ def tearDown(self):
             destroy_fun = cudashm.destroy_shared_memory_region
         for precreated_shm_region in self.precreated_shm_regions_:
             destroy_fun(precreated_shm_region)
+        super().tearDown()
```

Comment:
Why do you have to explicitly call the super here, but not for setUp?

Reply:
The class that I have written for collecting the test results doesn't need to use `setUp`. Result collection happens in the `run` method which is not explicitly called in subclasses. However, I need to ensure that `tearDown` is always called since storing the test result happens there.